packages/suite/src/types/suite/index.ts

```diff
@@ -110,7 +110,7 @@ export interface Dispatch extends ThunkDispatch<AppState, any, Action> {
 
 export type GetState = () => AppState;
 
-export type Store = ReduxStore<AppState, Action>;
+export type Store = ReduxStore<AppState, Action | UnknownAction>;
```

Comment:
Doesn't this weaken our Redux types a lot?

Reply:
Didn't notice. This fixes actually the typing tho. But will check. Good point.